src/.clang-format

```diff
@@ -0,0 +1,12 @@
+Language: Cpp
+BreakBeforeBraces: Attach
+PointerAlignment: Right
+TabWidth: 4
+IndentWidth: 4
+AccessModifierOffset: 0
+ColumnLimit: 120
+NamespaceIndentation: All
+AlignTrailingComments: true
+AllowAllParametersOfDeclarationOnNextLine: true
+AlwaysBreakTemplateDeclarations: true
+UseTab: Always
```

Comment:
```
IndentWidth: 4
ColumnLimit: 100
SortIncludes: false
AlignEscapedNewlinesLeft: false
SpacesBeforeTrailingComments: 1
```

Is the contents of the current file. I thought we had it in the repo, but it appears we don't.

We're not using tabs in this project